Program.cs

```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
+                case 2: TypeWriter.WriteLine(new Text($"the {monster.spices} "),
+                                             new Text("choped your head off ", ConsoleColor.DarkRed),
+                                             new Text("and took it as a trophy")); break;
+                case 3: TypeWriter.WriteLine(new Text($"The {monster.spices}" ),
+                                             new Text("got you with such a mean "),
+                                             new Text("left hook ", ConsoleColor.DarkRed),
+                                             new Text("that you died")); break;
+                case 4: TypeWriter.WriteLine(new Text($"The {monster.spices} "),
+                                             new Text("made shuch a scary face ", ConsoleColor.DarkRed),
+                                             new Text("that you got a heart attack")); break;
+                case 5: TypeWriter.WriteLine(new Text($"The {monster.spices} stepped on you and was so heavy that you were "),
+                                             new Text("instantly squished", ConsoleColor.DarkRed)); break;                                                          
+            }
+
+        }
+
+        private void fightDescriptionWin( Monster monster )
+        {
+            
+            if (knightOrMage == "m")
+            {
+                int winDescription = new Random().Next(1,4);
+                            switch ( winDescription )
+                            {
+                                case 1: TypeWriter.WriteLine(new Text($"You shoot a "),
+                                                             new Text("lightning bolt ", ConsoleColor.DarkCyan),
+                                                             new Text($"at the {monster.spices}, making them explode")); break;
+                                case 2: TypeWriter.WriteLine(new Text($"You summon a "),
+                                                             new Text("fireball ", ConsoleColor.DarkCyan),
+                                                             new Text("which turns them to ash")); break;
+                                case 3: TypeWriter.WriteLine(new Text("You "),
+                                                             new Text("freeze ", ConsoleColor.DarkCyan),
+                                                             new Text($"the {monster.spices} in a block of ice")); break;
+                            }
+            }
+            else if (knightOrMage == "k")
+            {
+                int winDescription = new Random().Next(1,4);
+                            switch ( winDescription )
+                            {
+                                case 1: TypeWriter.WriteLine(new Text("With one mighty blow you "),
+                                                             new Text("decapatated them")); break;
```

Comment:
decapitated

Reply:
them? ( that is plural ) ... how about "it" .. or the {monster.spieces}

Reply:
I just used {monster.spieces}